src/Repository/Jecoute/DataSurveyRepository.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace AppBundle\Repository\Jecoute;
+
+use AppBundle\Entity\Jecoute\DataSurvey;
+use Doctrine\Bundle\DoctrineBundle\Repository\ServiceEntityRepository;
+use Symfony\Bridge\Doctrine\RegistryInterface;
+
+class DataSurveyRepository extends ServiceEntityRepository
+{
+    public function __construct(RegistryInterface $registry)
+    {
+        parent::__construct($registry, DataSurvey::class);
+    }
+
+    public function countByEmailAnsweredForOneMonth(string $email, \DateTime $date): int
+    {
+        return $this
+            ->createQueryBuilder('dataSurvey')
+            ->select('COUNT(dataSurvey.id)')
+            ->andWhere('dataSurvey.emailAddress = :email')
+            ->andWhere('dataSurvey.postedAt >= :from')
+            ->andWhere('dataSurvey.postedAt < :until')
+            ->setParameter('email', $email)
+            ->setParameter('from', $date->modify('-1 month'))
+            ->setParameter('until', $date)
```

Comment:
if `$date` is not immutable date here you will have the same date as for `:from` parameter. I think you should clone the date, wdyt?